DFC.FindACourseClient/Repositories/CosmosRepository.cs

```diff
@@ -21,18 +21,66 @@ public CosmosRepository(CourseSearchAuditCosmosDbSettings cosmosDbConnection, ID
         {
             this.cosmosDbConnection = cosmosDbConnection;
             this.documentClient = documentClient;
+
+            CreateDatabaseIfNotExistsAsync().GetAwaiter().GetResult();
```

Comment:
Not a good idea to execute stuff in constructor, need separate initalise and should be called as part of registration

Reply:
Moved to DI